Source/Diagnostics/ReducedDiags/MultiReducedDiags.H

```diff
@@ -0,0 +1,41 @@
+#ifndef WARPX_DIAGNOSTICS_REDUCEDDIAGS_MULTIREDUCEDDIAGS_H_
+#define WARPX_DIAGNOSTICS_REDUCEDDIAGS_MULTIREDUCEDDIAGS_H_
+
+#include "ReducedDiags.H"
+#include <vector>
+#include <string>
+#include <memory>
+
+/**
+ * This class holds multiple instances of ReducedDiagnostics, and contains
+ * general functions to initialize, compute, and write these diagnostics
+ * to file.
+ */
+class MultiReducedDiags
+{
+public:
+
+    /// Bool: whether or not reduced diagnostics are activated
+    int m_plot_rd = 0;
+
+    /// names of reduced diagnostics
+    std::vector<std::string> m_rd_names;
```

Comment:
This is just a suggestion: I am not sure `MultiRedicedDiags` needs to know have the name of all diagnostics as a member variable. It seems that the number of diagnostics would suffices. But that's a very minor point.

Reply:
Let's not care about that for now.